### PR TITLE
translate resource-reservation.md

### PR DESCRIPTION
**Pull request by forcoder**
translate resource-reservation.md

### CODE REVIEW

File: i18n/zh-Hans/docusaurus-plugin-content-docs/current/user-manuals/resource-reservation.md [hunk 1]
```diff
@@ -1,43 +1,42 @@
...
 
-Resource Reservation is an ability of koord-scheduler for reserving node resources for specific pods or workloads.
+资源预留是koord-scheduler的一种为某些特定pod或负载预留节点资源的能力。
 
-## Introduction
+## 介绍
 
-Pods are fundamental for allocating node resources in Kubernetes, which bind resource requirements with business logic.
-However, we may allocate resources for specific pods or workloads not created yet in the scenarios below:
+容器是kubernetes节点资源分配的基础载体，他根据业务逻辑绑定对应的资源需求。但是我们可能分为一些还没创建的特定容器和负载分配资源，例如：
```
[the remark this answers is not in the document]
Comment (forcoder) [reply i18n/zh-Hans/docusaurus-plugin-content-docs/current/user-manuals/resource-reservation.md:7]: 收到

[the remark this answers is not in the document]
Comment (forcoder) [reply i18n/zh-Hans/docusaurus-plugin-content-docs/current/user-manuals/resource-reservation.md]: 已修改

**Comment by saintube** [comment]
/lgtm
PTAL @eahydra

**Comment by eahydra** [comment]
Thanks @forcoder 
/lgtm
/approve

**Review by eahydra — APPROVED** [review]

**Pull request merged**